src/pages/ContactUsPage/index.tsx

```diff
@@ -1,28 +1,44 @@
-import React, { FormEvent } from 'react';
+import React, { FormEvent, useState } from 'react';
 import Button from 'components/Common/Button';
 import './index.css';
+import { BEGANTOFLY4_GMAIL_EMAIL_API_KEY } from 'configs/env.config';
 
 const ContactUsPage: React.FC = () => {
-  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+  const [submitted, setSubmitted] = useState(false);
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
     event.preventDefault();
     // TODO: Handle form submission
+    const formData = new FormData(event.currentTarget);
+
+    const response = await fetch('https://api.web3forms.com/submit', {
+      method: 'POST',
+      body: formData,
+    });
+
+    if (response.ok) {
+      setSubmitted(true);
+    } else {
+      console.error('Form submission error:', response.statusText);
```

Comment:
Hello sir, possible to rectify this linting issue? Either ignore or handle differently ba